examples/images_contours_and_fields/interpolation_none_vs_nearest.py

```diff
@@ -1,62 +0,0 @@
-"""
```

Comment:
I completely agree with your assessment that we have too many examples and that many are poorly designed.  However this specific example is exactly addressing a very specific question (the apparent duplication of interpolation methods between "none" and "nothing").  Thus my suggestion to move it to a FAQ entry.